src/HospitalLibrary/Rooms/Dtos/RoomSearchDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalLibrary.Rooms.Dtos
+{
+    public  class RoomSearchDTO
+    {
+        
+         public string RoomType { get; set; }
```

Comment:
room type is enum, so this is not valid, if you dont know how to map enums check registration blood type